packages/cms/plugin/index.ts

```diff
@@ -557,7 +571,7 @@ class CmsPlugin {
                           firstName: 'required',
                           lastName: 'required',
                           email: 'required|email',
-                          password: 'required|min:12',
```

Comment:
Are we deleting this very important validation rule ?

Reply:
Yes, that's because it wasn't being captured.
Even if it's put there, it wasn't going to be active